README.md

```diff
@@ -243,7 +243,7 @@ topic.publish({
 }, function(err) {});
 
 // Subscribe to the topic.
-topic.subscribe('new-subscription', function(err, subscription) {
+topic.subscribe('new-subscription', { reuseExisting: true }, function(err, subscription) {
```

Comment:
Maybe we should rename the subscription, because it's not "new" if the user is worried about it already existing.

How about

```js
var options = {
  reuseExisting: true 
};

topic.subscribe('subscription-name', options, function(err, subscription) {
```